example/nested/outter.yaml

```diff
@@ -0,0 +1,37 @@
+# Directory structure:
+# ├── Makefile
+# ├── outter.yaml
+# ├── private
+# │   ├── judge.yaml
+# │   ├── supplied
+# │   │   └── <supplied code>
+# │   └── testcases
+# │       └── <test case>
+# └── submit
+#     └── <id>
+#         └── src
+#             └── <submited code>
```

Comment:
Is this draw with `tree` command?

Reply:
This was draw with `tree` command and been manually modified